test/upload_test.js

```diff
@@ -27,6 +27,14 @@ describe('S3 Webpack Upload', () => {
         .then(assertFileMatches);
     });
 
+    it('uploads entire directory with alternate pathing s3', () => { // eslint-disable-line
+      config = testHelpers.createWebpackConfigAlt({ s3Config });
```

Comment:
Why not put this test within its own `describe` function as for now `config` param shadows previously generated config?

Reply:
Moved to its own describe function